CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Minor changes
 
 -   **terra-node-tree** add support for mat-icons.
+-   **tooltip** added `TerraTooltipModule` to be able to individually include tooltip functionality.
```

Comment:
```suggestion
-   **tcTooltip** added `TerraTooltipModule` to be able to individually include tooltip functionality.
```

Reply:
it should be tcTooltip since its the selector of the tooltip and we always use the selector in there somehow...